tests/e2e/test_navigation_page_non_student.py

```diff
@@ -63,7 +63,7 @@ def test_ta(self):
                                 .find_element_by_id('graded_tbody')
                                 .find_elements_by_class_name("gradeable_row")))
 
-        self.assertEqual(7, len(self.driver.find_element_by_class_name(
+        self.assertEqual(4, len(self.driver.find_element_by_class_name(
```

Comment:
Why is this being decreased?

Reply:
Graders / TAs were being shown 3 empty columns all the time from the instructor-only buttons. This just cleans that up.